app/fixtures/sample.json

```
@@ -0,0 +1,26 @@
+[
+  {
+    "model": "app.TableMetaData",
+    "pk": 1,
+    "fields": {
+      "name": "TestTable",
+      "original_file_name": "OriginalFileName.csv",
+      "description": "Description for a table",
+      "created_at": "2024-01-22 11:16:13.000Z",
+      "modified_at": "2024-01-22 11:16:13.000Z"
+    }
+  },
+  {
+    "model": "app.ColumnMetaData",
+    "pk": 1,
+    "fields": {
+      "table_metadata_id": 1,
+      "name": "text_column",
+      "title": "Text Column",
+      "description": "Description of column",
+      "data_type_id": 1,
```

```suggestion
      "data_type": 1,
```
Not sure whether that's changed in a PR somewhere, but on `main` this field is called `data_type` not `data_type_id`

---

data_type becomes data_type_id in the database (it references a data_type by the id of the data_type). Hence, data_type_id is also valid

---

Ah, I see!